demos/modules/demo_image.mjs

```diff
@@ -279,3 +280,19 @@ function genSlide04(pptx) {
 	);
 	slide.addImage({ path: IMAGE_PATHS.sydneyBridge.path, x: 0.5, y: 5.16, h: 1.8, w: 12.33 });
 }
+
+/**
+ * SLIDE 5: Image Shadow
+ * @param {PptxGenJS} pptx
+ */
+ function genSlide05(pptx) {
+	let slide = pptx.addSlide({ sectionTitle: "Images" });
+
+	slide.addTable([[{ text: "Image with Shadow Props", options: BASE_TEXT_OPTS_L }, BASE_TEXT_OPTS_R]], BASE_TABLE_OPTS);
+	slide.addNotes("API Docs: https://gitbrent.github.io/PptxGenJS/docs/api-images.html");
+	slide.slideNumber = { x: "50%", y: "95%", w: 1, h: 1, color: COLOR_BLUE };
+
+	// EXAMPLES
+	slide.addText("shadow: {type: 'outer', opacity: 0.5, blur: 20, color: '000000', offset: 20, angle: 270}", { x: 0.5, y: 0.6, w: 6.0, h: 0.3, color: COLOR_BLUE });
```

Comment:
<img width="450" alt="Screenshot 2022-08-10 at 16 58 44" src="https://user-images.githubusercontent.com/10131999/183937666-77e401fa-ec18-46ba-a7ca-51ab882b928c.png">
 
He uses a convention here i.e. Shadow: \`type:outer, opacity:0.5`

Reply:
It may be useful to have an example for each shadow type